src/common/transformations/tests/op_conversions/convert_avgpool_downgrade.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_avgpool_downgrade.hpp"
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/ov_test_utils.hpp"
+#include "openvino/op/avg_pool.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "openvino/opsets/opset14.hpp"
```

Comment:
Avoid including whole opsets. It causes significant object file size bloat, leading to compilation termination in some cases. Including only used ops should be enough.